crates/oci-distribution/src/client.rs

```diff
@@ -765,4 +1096,183 @@ mod test {
                 .is_err());
         }
     }
+
+    #[tokio::test]
+    async fn can_push_layer() {
+        let mut c = Client::new(ClientConfig {
+            protocol: ClientProtocol::Http,
+        });
+        let url = "oci.registry.local/hello-wasm:v1";
```

Comment:
so in order for this test to pass, you have to spin up an OCI registry and serve it on `oci.registry.local`, correct?

If so perhaps we should add a `skip` admonition and re-introduce this as a TODO item. We have some CI infrastructure in place where we can test this, but we'd have to verify credentials and what would be a good namespace to push webassembnly modules to.

Reply:
That is correct, I figured that it would break any tests unless a registry was already spun up in your CI. I'm a fan of adding a way to skip these tests with a TODO.

Reply:
For the three local registry tests, I added the `#[ignore]` attribute so they aren't run automatically with `cargo test`